ui/src/utils/tests/msw/handlers.ts

```diff
@@ -108,12 +108,10 @@ const handlers = [
 
         /* Inspect request */
         const requestBody = await request.arrayBuffer()
-        const decodedRequest = msgpack.decode(
-          new Uint8Array(requestBody),
-        ) as algosdk.modelsv2.SimulateRequest
+        const decodedRequest = msgpack.decode(new Uint8Array(requestBody)) as SimulateRequest
         // console.log('decodedRequest', decodedRequest)
 
-        const txns = decodedRequest.txnGroups[0].txns
+        const txns = decodedRequest['txn-groups'][0].txns
```

Comment:
wait - what? it went from a structured type to unstructured in newer sdk/clients?
that seems like a regression.

Reply:
These Mock Service Worker handlers are directly intercepting API requests and mocking responses. Those still use kebab-case properties. Reverting this to the original `txn-groups` fixes a test that was failing because I mistakenly updated it to camelCase like I had been everywhere else.

Reply:
e.g.
<img width="446" alt="image" src="https://github.com/user-attachments/assets/177bca0f-9b5b-4e1d-b158-4929c81ec537">